airbyte-integrations/connectors/source-hardcoded-records/source_hardcoded_records/run.py

```diff
@@ -1,13 +1,27 @@
 # Copyright (c) 2024 Airbyte, Inc., all rights reserved.
-
-
 import sys
 
+import ddtrace
+
 from airbyte_cdk.entrypoint import launch
 from source_hardcoded_records import SourceHardcodedRecords
 
 
-def run():
+# Automatically patch all supported libraries to enable tracing
+ddtrace.patch_all()
+
+# Set DataDog configuration options directly in your Python code
+ddtrace.config.env = "dev"                          # Equivalent to DD_ENV
+ddtrace.config.service = "source-hardcoded-records" # Equivalent to DD_SERVICE
+ddtrace.config.version = "1.0.0"                    # Equivalent to DD_VERSION
+ddtrace.config.agent_hostname = "localhost"         # Equivalent to DD_AGENT_HOST
```

Comment:
@davinchia - Do you know if the container (or pod) might already have the DataDog agent installed and if so, if the env vars would already be set correctly?